app/helpers/metadata_helper.rb

```diff
@@ -253,7 +248,7 @@ def validate_metadata_csv_for_samples(samples, metadata, new_samples = false)
           error_aggregator.add_error(:sample_not_found, [index + 1, sample.name])
         end
 
-        if val_errors[:invalid_key_for_host_genome].present?
+        if val_errors[:invalid_host_genome].present?
```

Comment:
was this a bug being fixed? and it seems like `invalid_key_for_host_genome` is a better name, so maybe rename where `val_errors` is updated instead.

Reply:
Good call.